### DISCUSSION TITLE
Keyboard types wrong characters

### DISCUSSION THREAD

**Discussion by potatoleek** [SpiralLinux/SpiralLinux-project#136] [Q&A] [answered]
While testing SpiralLinux 11.220925 XFCE on an Acer Aspire 4733Z, I ran into a strange issue with the keyboard (US English): Many of the keys do not produce the right characters.

Examples (key pressed followed by output):
0 /
) /
u 4
i 5
o 6
j 1
k 2
l 3

I checked the XFCE settings and confirmed it's using the English US keyboard layout. The keyboard works fine in Linux Mint 21 XFCE.

**Comment by potatoleek** [comment, marked as answer]
This has been solved. It seems my laptop has some kind of virtual number pad on top of other keys. After toggling the NumLock, I was able to type the correct characters.

**Reply by janxkoci** [reply]
Same as https://github.com/orgs/SpiralLinux/discussions/92
